com.spotify.Client.json

```diff
@@ -105,6 +105,21 @@
                 }
             ]
         },
+        {
+            "name": "spotifywm",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make",
```

Comment:
`-j $FLATPAK_BUILDER_N_JOBS`

Reply:
I'm not sure there's any point to that; spotifywm is a single .cpp file.

Reply:
No reason not to, common convention.

Reply:
Okay, updated. Since `/bin/sh` parses it, I did `${FLATPAK_BUILDER_N_JOBS:-1}` in case the variable disappears or is unset for some reason. Is that okay?